kibana-notebooks/public/components/notebook.tsx

```diff
@@ -837,7 +819,7 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
                           description="Write contents directly using markdown, SQL or PPL"
                           footer={
                             <EuiButton onClick={() => this.addPara(0, '', 'CODE')} style={{ marginBottom: 17 }}>
-                              Add
+                              Add markdown paragraph
```

Comment:
This should be `code paragraph`?

Reply:
UX screens just have `Add`, accidentally reverted this one with other changes. Re-adding